alacritty.yml

```diff
@@ -63,6 +63,13 @@
   #     - buttonless: Title bar, transparent background and no title bar buttons
   #decorations: full
 
+  # Background opacity
+  #
+  # Window opacity as a floating point number from `0.0` to `1.0`.
+  # The value `0.0` is completely transparent and `1.0` is opaque.
+  #background_opacity: 1.0
```

Comment:
I wonder if it's worth it to change this to just `opacity`? I think `window.opacity` should be pretty clear by itself, that way we could take this opportunity to make things a little more concise.